javascript-functions/main.js

```diff
@@ -0,0 +1,43 @@
+// convertMinutesToSeconds(minutes)
+function convertMinutesToSeconds(minutes) {
+  return minutes * 60;
+}
+console.log(
+  'Value of function convertMinutesToSeconds w/ argument 8:',
+  convertMinutesToSeconds(8)
+);
+
+// greet(name)
+function greet(name) {
+  return 'Hey, ' + name;
+}
+console.log('Value of function greet w/ argument Jesus:', greet('Jesus'));
+
+// getArea(width, height)
+function getArea(width, height) {
+  return width * height;
+}
+console.log('Value of function getArea w/ argument of 4, 36:', getArea(4, 36));
+
+// getFirstName(person)
+function getFirstName(person) {
+  person = {
+    firstName: 'Jesus',
+    lastName: 'Araujo',
+  };
```

Comment:
You don't want to define `person` inside of this function, you want to pass this object when you call the function

Reply:
<img width="291" alt="Screenshot 2023-05-09 at 7 27 56 PM" src="https://github.com/j-alejandro-araujo/c0523-code-solutions/assets/91649194/d3fe5c2a-64fe-4136-a98f-fe4c2f4e304e">

Fixed, good catch!